### PR TITLE
Albums list

### PR DESCRIPTION
**Pull request by dbenavidesv**
## Summary
- Added graphql query for getting the albums list
- The albums list can be paginated and sorted using the query params: `offset`, `limit`, `orderBy`  
- Refactored the fetching of photos to do it by type resolvers
- Fixed issues related to the usage of `this` and DRY compliance

## Trello Card

https://trello.com/c/WTnkKHZD

### CODE REVIEW

File: app/graphql/albums/resolvers.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+const logger = require('../../logger');
+const albumsService = require('../../services/albums');
+
+exports.getAlbum = (root, args) => {
```
Comment (gonziesc) [inline app/graphql/albums/resolvers.js]: destructure args

File: app/graphql/albums/resolvers.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  logger.info(`Fetching album with id: ${args.id}`);
+  return albumsService.getAlbum(args.id).then(album => ({
+    ...album,
+    artist: album.userId
+  }));
+};
+
+exports.getAlbums = (root, args) => {
```
Comment (gonziesc) [inline app/graphql/albums/resolvers.js]: same here

**Review by gonziesc — CHANGES_REQUESTED** [review]

File: app/graphql/albums/resolvers.js (hunk 1 above)
Comment (dbenavidesv) [reply app/graphql/albums/resolvers.js]: Got it.

File: app/graphql/albums/resolvers.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+exports.getAlbum = (root, { id }) => {
+  logger.info(`Fetching album with id: ${id}`);
+  return albumsService.getAlbum(id).then(album => ({
+    ...album,
+    artist: album.userId
+  }));
+};
+
+exports.getAlbums = (root, { offset, limit, orderBy }) => {
```
Comment (gonziesc) [inline app/graphql/albums/resolvers.js:12]: orderBy?

Comment (dbenavidesv) [reply app/graphql/albums/resolvers.js:12]: Sorry, I don't understand, what's wrong here?

Comment (gonziesc) [reply app/graphql/albums/resolvers.js:12]: nothing, i missundestood something

**Review by gonziesc — APPROVED** [review]

**Pull request merged**